applications/ParticleMechanicsApplication/tests/test_particle_erase_process.py

```diff
@@ -59,6 +60,12 @@ def _create_elements(self, initial_mp):
         initial_mp.CreateNewElement("UpdatedLagrangian3D8N", 1, [1,2,3,4,5,6,7,8], initial_mp.GetProperties()[1])
         KratosMultiphysics.VariableUtils().SetFlag(KratosMultiphysics.ACTIVE, True, initial_mp.Elements)
 
+    def _create_conditions(self, initial_mp):
+        initial_mp.CreateNewCondition("Condition3D4N", 1, [2,4,8,6], initial_mp.GetProperties()[1])
+        KratosMultiphysics.VariableUtils().SetFlag(KratosMultiphysics.BOUNDARY, True, initial_mp.Conditions)
+        for condition in initial_mp.Conditions:
```

Comment:
use variable utils

Reply:
Correct me if I am wrong, but SetScalarVar function in VariableUtils only work with double?
https://github.com/KratosMultiphysics/Kratos/blob/fd097ab98a51c196fc61a62717c3939b07379f69/kratos/python/add_utilities_to_python.cpp#L263

Reply:
could be :)